iohk-monitoring/src/Cardano/BM/Data/Tracer.lhs

```diff
@@ -150,19 +179,23 @@ instance ToObject Foo
 \end{spec}
 
 \begin{code}
-class ToJSON a => ToObject a where
-    toObject :: a -> Object
-    default toObject :: a -> Object
-    toObject v = case toJSON v of
+class ToObject a where
+    toObject :: TracingVerbosity -> a -> Object
+    default toObject :: ToJSON a => TracingVerbosity -> a -> Object
+    toObject _ v = case toJSON v of
         Object o     -> o
         s@(String _) -> HM.singleton "string" s
         _            -> mempty
 
+mkObject :: ToObject a => [(Text, a)] -> HM.HashMap Text a
+mkObject = HM.fromList
```

Comment:
Do we really need an alias for `HM.fromList`? :-)

Reply:
it's not an alias as the types here are more specific.